src/Cardano/Wallet/Api.hs

```diff
@@ -24,8 +42,11 @@ type Api = Wallets
 type Wallets =
     DeleteWallet
     :<|> GetWallet
+    :<|> ListAddresses
```

Comment:
The idea was to have groups matching the groups from the API specification :sweat_smile: 
So I'd suggest to have:

```hs
type Wallets =

type Wallets =
    DeleteWallet
    :<|> GetWallet
    :<|> ListWallets
    :<|> PostWallet
    :<|> PutWallet
    :<|> PutWalletPassphrase

type Addresses =
    listAddresses
```

Reply:
Interesting! I was under the impression that the `listAddresses` operation was within the `wallets` group:

```
  /wallets/{walletId}/addresses:                                                                                                                                                      
    get:                                                                                                                                                                              
      operationId: listAddresses      
```
Should we update the specification to move it to another group?

Reply:
Nope, the spec is correct here. Endpoints are grouped by `tags`, cf:

https://github.com/input-output-hk/cardano-wallet/blob/master/specifications/api/swagger.yaml#L896

https://github.com/input-output-hk/cardano-wallet/blob/master/specifications/api/swagger.yaml#L844

So, even if, conceptually, the `Address` resource is a sub-resource of the `Wallet` resource, operations related to addresses are in a separate group to be easier to identify (same apply for transactions).

Reply:
Okay! I've moved `ListAddresses` to a separate group.